fs/unionfs/fs_unionfs.c

```diff
@@ -1285,6 +1325,45 @@ static int unionfs_fstat(FAR const struct file *filep, FAR struct stat *buf)
   return ops->fstat ? ops->fstat(&uf->uf_file, buf) : -EPERM;
 }
 
+/****************************************************************************
+ * Name: unionfs_fchstat
+ *
+ * Description:
+ *   Change information about an open file associated with the file
+ *   descriptor 'fd',.
+ *
+ ****************************************************************************/
+
+static int unionfs_fchstat(FAR const struct file *filep,
+                           const FAR struct stat *buf, int flags)
```

Comment:
```suggestion
                           FAR const struct stat *buf, int flags)
```
nit: just to keep consistency on the order of the qualifiers

Reply:
Done.